docs/DeveloperGuide.md

```diff
@@ -549,15 +556,24 @@ testers are expected to do more *exploratory* testing.
     1. Test case: `find tag/marketing sales`.<br>
        Expected: Passengers are shown are same as previous. Status message shows tag provided is invalid.
 
+### Finding a pool
 
-### Saving data
+1. Finding a pool while all pools are being shown.
 
-1. Dealing with missing/corrupted data files.
+    1. Prerequisites: Multiple pools in the list. List all pools using the `list` command.
 
-    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+    1. Test case: `findPool n/lenny`.<br>
```

Comment:
is the tester expecting exactly these specific people? `Alan Poh`, `Lenny Hoon`, and `Turner Peck`
What if the tester has other names in the list instead

Reply:
These tests are provided on the assumption that the user is testing with the default addressbook.json that is created by the GME system. This also addresses your other comment regarding the test case. I will fix this by adding this assumption in the prerequisites section of the testcases.